doc/code/qml_pauli.rst

```diff
@@ -151,3 +152,18 @@ the groups of observables:
 
 For a larger example of how grouping can be used with PennyLane, check out the
 `Measurement Optimization demo <https://pennylane.ai/qml/demos/tutorial_measurement_optimize/>`_.
+
+Dynamical Lie Algebras
+----------------------
+
+Dynamical Lie Algebra (DLA) functionality in PennyLane.
+
+See our `introduction to Dynamical Lie Algebras for quantum practitioners <https://pennylane.ai/qml/demos/tutorial_liealgebra/>`__.
+
+.. currentmodule:: pennylane.pauli.dla
+
+.. autosummary::
+    :toctree: api
+
+    ~PauliVSpace
+    ~lie_closure
```

Comment:
Please could we get it so that `lie_closure` points to the top-level version, i.e., so you'd see `qml.lie_closure` rather than `qml.pauli.dla.lie_closure`? The preference is to have a single (preferably top level) entrypoint that is recommended to users in the docs.